API/Controllers/SeatsController.cs

```diff
@@ -31,11 +31,27 @@ public IEnumerable<Seats> GetAllSeats()
         //    return _seatsBLL.GetSeatById(id);
         //}
 
-        // list me all the free bus seats
+        // list me all the free/reserved bus seats
         [HttpGet("{id}/{status}")]
         public IEnumerable<Seats> GetSeatsByBus(int id, string status)
         {
             return _seatsBLL.GetSeatsByBus(id, status);
         }
+
+        [HttpPost]
+        public IActionResult Index([FromBodyAttribute] Seats seat)
```

Comment:
Rename the method to something more appropriate, Index does explain anything here. i.e. CreateNew